packages/ui-docs-client/src/compileMarkdown.js

```diff
@@ -263,9 +265,7 @@ function createRenderer() {
             code: matter.content,
             render: (_el) => {
               el = theme ? (
-                <ApplyTheme theme={ApplyTheme.generateTheme(theme)}>
-                  {_el}
-                </ApplyTheme>
+                <EmotionThemeProvider theme={theme}>{_el}</EmotionThemeProvider>
```

Comment:
Sadly this won't work, because `const theme = (matter.data || {}).theme` will be a string (name of the theme, if it is given), and the theme provider needs an object to work. I wrote about this issue on the slack channel, we should discuss it, but I guess it works for now (there are other issues we need to fix too).

Reply:
As per our discussion we will fix this in a future PR